src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx

```diff
@@ -41,7 +41,7 @@ export const AdminSpaceCard: React.FC<AdminSpaceCardProps> = ({ venue }) => {
         Manage Space
       </ButtonNG>
       <ButtonNG linkTo={adminNGSettingsUrl(venue.id)} variant="primary">
-        Advanced Settings
+        Space Settings
```

Comment:
I know it is crude solution, but at least it allows us to easily list all uses of the term and update accordingly.

```suggestion
        {SPACE_TAXON.capital} Settings
```

Reply:
done 40ab2db